src/main/kotlin/com/criptext/mail/push/data/LinkDeviceActionService.kt

```diff
@@ -0,0 +1,66 @@
+package com.criptext.mail.push.data
+
+import android.app.IntentService
+import android.app.NotificationManager
+import android.content.Context
+import android.content.Intent
+
+
+class LinkDeviceActionService : IntentService("Link Device Action Service") {
+
+    companion object {
+        const val APPROVE = "Approve"
+        const val DENY = "Deny"
+
+        private var pushDataSource: PushDataSource? = null
+        private var randomId: String = ""
```

Comment:
why do you need to keep this data here? the Service should be as stateless as possible.